src/os/bluestore/bluestore_tool.cc

```diff
@@ -681,7 +681,8 @@ int main(int argc, char **argv)
     // kludge some things into the map that we want to populate into
     // target dir
     label.meta["path_block"] = devs.front();
-    label.meta["type"] = "bluestore";
+    //now type is stored in label, this allows bluestore-rdr
+    //label.meta["type"] = "bluestore";
```

Comment:
OK, so now we're leaving the key intact. Is there any business in  checking whether it's `buestore` or `bluestore-rdr`?

Reply:
There is - we do not want an accidental upgrade from pacific to quincy-rdr.